src/main/java/moneytransferapp/InitiateMoneyTransfer.java

```diff
@@ -0,0 +1,39 @@
+package moneytransferapp;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import java.util.UUID;
+
+// @@@SNIPSTART money-transfer-project-template-java-workflow-initiator
+public class InitiateMoneyTransfer {
+
+    public static void main(String []args) throws Exception {
+        // WorkflowServiceStubs is a gRPC stubs wrapper
+        // That talks to the local Docker instance of the Temporal server.
+        WorkflowServiceStubs service = WorkflowServiceStubs.newInstance();
+        WorkflowOptions options = WorkflowOptions.newBuilder()
+                .setTaskQueue(Shared.MONEY_TRANSFER_TASK_QUEUE)
+                .setWorkflowId("money-transfer-workflow-001")
+                .build();
+        // WorkflowClient can be used to start, signal, query, cancel, and terminate Workflows.
+        WorkflowClient client = WorkflowClient.newInstance(service);
+        // WorkflowFlowStub converts the parameters that are passed to it so that they can be passed to the server.
+        MoneyTransferWorkflow workflow = client.newWorkflowStub(MoneyTransferWorkflow.class, options);
+
+        String referenceId = UUID.randomUUID().toString();
```

Comment:
Why do you go to the trouble of generating a random id? For multiple runs?

Reply:
Yes, carried this over from the java sample - and yes, if you rm the WorkflowId you can run this code multiple times.
I am wondering if we should specify the WorkflowId here or not - and call that out in the tutorial at all...

Reply:
Added comment to WorkflowId, which makes this more rational.